documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
 
-* node name: is anonymized by default always
-* resource type name: the core types that are built-in to Puppet are not anonymized, including some common types from the modules: stdlib, postgresql, rabbitmq, puppetdb, apache, mrep, f5, apt, registry, concat and mysql. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type names as they may contain confidential information.
-* resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service and Stage.
-* parameter names: are never anonymized
-* parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit` and `schedule`.
-* report log messages: are always anonymized
-* file names: are always anonymized
-* file numbers: are left as they are
-* log messages: are always anonymized
-* metrics: metric names in the `time` category are anonymized as resource
-  types.
+* Node name: is anonymized by default.
+* Resource type name: the core types that are built into Puppet are not anonymized, including some common types from the modules: `stdlib`, `postgresql`, `rabbitmq`, `puppetdb`, `apache`, `mrep`, `f5`, `apt`, `registry`, `concat`, and `mysql`. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type name, as these may contain confidential information.
+* Resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service, and Stage.
+* Parameter names: are never anonymized.
+* Parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit`, and `schedule`.
+* Report log messages: are always anonymized.
+* File names: are always anonymized.
+* File numbers: are left as they are.
+* Log messages: are always anonymized.
+* Metrics: metric names in the `time` category are anonymized as resource types.
 
 ### Profile: low
 
-This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. By default most things are not anonymized except for:
+This profile is aimed at hiding security information specifically, but leaving most of the data in its original state. However, by default, the following categories are annonymized:
 
-* node name: is always anonymized
-* parameter values: we specifically anonymize any values and messages for any parameter name containing the strings: password, pwd, secret, key, private. Everything else is left alone.
-* log messages: are always anonymized
+* Node name: is always anonymized.
+* Parameter values: this profile will anonymize values and messages for any parameter name containing the strings `password`, `pwd`, `secret`, `key`, or `private`. Everything else is left alone.
```

Comment:
"this profile will" seems redundant in a bullet under the profile. maybe something like "Only values and messages for parameter names containing the strings ... are anonymized."